crates/sui-protocol-config/src/lib.rs

```diff
@@ -16,7 +17,7 @@ const MAX_PROTOCOL_VERSION: u64 = 3;
 //
 // Version 1: Original version.
 // Version 2: Framework changes, including advancing epoch_start_time in safemode.
-// Version 3: gas model v2, including all sui conservation fixes.
+// Version 3: gas model v2, including all sui conservation fixes, limits on `max_size_written_objects`, `max_size_written_objects_system_tx`
```

Comment:
similar formatting nits in this file, comments often get to 150-200 cols, can you make these fit in my 100 col macbook where possible/reasonable? :)